documentation/concepts/transports.md

```diff
@@ -1,3 +1,27 @@
 ```yaml
 title: Transports
 ```
+
+Machines, within IoT systems, communicate amongst each other by exchanging
```

Comment:
Strike ',within IoT systems, '

I think that 'Machines' or 'Connected machines and devices' is a cleaner open